src/data/faq.json

```
@@ -1789,9 +1817,13 @@
                                 "anchor": "hearing_impaired",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated ?????r</b>",
+                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ?????h.",

```

Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h.",

```